### ISSUE TITLE
How to train multi-resolution 24-net and 48-net?

### ISSUE THREAD

**Issue by hiyijian** [anson0910/CNN_face_detection#17]
hi @anson0910 ,
  You said "Multi-resolution is not used for simplicity, you can add them in the .prototxt files under CNN_face_detection_models to do so" in readme. It seems that it is very easy to add this feature.  Would you please to explain it more clear?

**Comment by anson0910** [comment]
I believe this can be done by the following steps:

1. Add the 12-net layers to the 24-net .prototxt files (concatenating the fully connected layers as shown in the paper)
2. Transplant the trained 12-net parameters to the multi-resolution net, and set the local learning rate to 0, as shown [here](https://groups.google.com/forum/#!msg/caffe-users/wfwl468OpG4/JlnnCcaEboYJ)
3. Train the net

**Comment by hiyijian** [comment]
Cool. I will have a shoot. Thank you
